src/player.cpp

```diff
@@ -159,7 +159,7 @@ void Player::Init(int argc, char *argv[]) {
 
 	// Create initial directory structure in our private area
 	// Retrieve save directory from persistent storage
-	EM_ASM(
+	EM_ASM(({
```

Comment:
The added braces (`(`) seem superfluous here, the curly braces should be enough.

Reply:
Unfortunately they are needed here because the Javascript code contains a "," :/

>  The C preprocessor does not have an understanding of JavaScript tokens, and as a result, if the code block contains a comma character ,, it may be necessary to wrap the code block inside parentheses. For example, code EM_ASM(return [1,2,3].length); will not compile, but EM_ASM((return [1,2,3].length)); does.